s3/pkg/service/service.go

```diff
@@ -522,6 +523,22 @@ func (s *s3Service) CountObjects(ctx context.Context, in *pb.ListObjectsRequest,
 	return nil
 }
 
+func (s *s3Service) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3, Null *pb.Null) error {
+	log.Info("backendCheck is called in s3 service.")
+	backendDetailtry := &backend.BackendDetail{}
```

Comment:
@vineela1999 why are we making a copy here ?

Reply:
because it doesnt accept BackendDetail of backend  its require to copy values of it to BackendDetailS3 of S3